src/data/faq.json

```diff
@@ -417,7 +417,7 @@
                         "textblock": [
                             "As of version 2.19 of the Corona-Warn-App, the assignment of certificates to persons is more fault-tolerant. Small differences in the spelling of first and last names are ignored so that the certificates are still automatically assigned to the same person.",
                             "The Corona-Warn-App can recognize that certificates belong to one person if the dates of birth match exactly and: <ul><li>the first and last names match exactly,</li><li>at least one first name and at least one last name match or</li><li>the first and last names are swapped.</li></ul>",
-                            "In case your name contains umlauts, they will be converted automatically. If you have a title of \"Dr.\", it will be ignored.",
+                            "In case your name contains umlauts, they will be converted automatically. The academic doctorate now no longer leads to the distinction of names.",

```

Comment:
"The academic doctorate now no longer leads to the distinction of names." 

I wonder why this was changed? It seems less easy to understand than the previous text. I'm not even sure that the term "academic doctorate" is accurate. Depending on a country's legal framework there is also a "professional doctorate" which allows the holder to title themselves "Dr.".

The sentence now sounds like a literal translation of the German "Der akademische Doktortitel führt nun nicht mehr zur Unterscheidung von Namen." which doesn't read well in English.

Let me know if you want a suggestion for improvement, otherwise I will leave it be.

Reply:
@MikeMcC399 thanks for your feedback. We would be happy to hear your suggestion.

Reply:
@larswmh 

I noticed a grammar issue in the German paragraph, regarding the sudden use of "you" in the middle of the text, so I'm also suggesting to change it in the English version:

"If a name contains umlauts, they will be converted automatically. A doctorate in a name will now be ignored for the purposes of comparing names.",

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2526/commits/3d76e6f74e60bd94c4530492103dd75055b0f04d, thanks for the suggestion